packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -411,7 +412,7 @@ export const startRewardDistributor = async ({
   const feeDistributorTerms = await deeplyFulfilledObject(
     harden({
       timerService,
-      collectionInterval: 60n * 60n, // 1 hour
+      collectionInterval: SECONDS_PER_HOUR,
```

Comment:
to my eye this would be more clear as:

```suggestion
      collectionInterval: 1n * SECONDS_PER_HOUR,
```